app/controllers/users/verify_account_controller.rb

```diff
@@ -8,9 +8,9 @@ def index
 
     def create
       @verify_account_form = build_verify_account_form
+
       if @verify_account_form.submit
-        flash[:success] = t('account.index.verification.success')
-        redirect_to account_path
+        redirect_to sign_up_completed_url
```

Comment:
Sorry that was worded really badly; but the idea is that the confirmation letter is going to direct the user to go back to the service provider's URL. It should be in the invision docs linked in the github issue